internal/testdata/states.go

```diff
@@ -108,28 +111,15 @@ func createOutcome(first types.Address, second types.Address, x, y uint) outcome
 	}}
 }
 
-func createOutcomeFromMap(amounts map[types.Address]uint) outcome.Exit {
-	var allocations []outcome.Allocation
-
-	// Generate a list of addresses from the map
-	addresses := make([]types.Address, 0, len(amounts))
-	for address, _ := range amounts {
-		addresses = append(addresses, address)
+func createLongOutcome(items ...simpleOutcome) outcome.Exit {
```

Comment:
What does `Long` mean? More than one entry?

Reply:
yes -- I can clarify this in a doctoring?

Reply:
https://github.com/statechannels/go-nitro/pull/448/commits/8086fb6bba3e05a3e86cbf60c8bc276336dbf4b6